cli.py

```diff
@@ -68,18 +69,26 @@ def main(log_level):
               type=str, help="D3A domain name")
 @click.option('-w', '--web-socket', default=DEFAULT_WEBSOCKET_DOMAIN,
               type=str, help="D3A websocket URL")
+@click.option('-j', '--json-file-name', type=str,
```

Comment:
I suggest using a name that would give information about the content of the file, rather than its format. Something like `--simulation-info` (or anything better, I can't think of anything else at the moment).